.travis.yml

```diff
@@ -2,3 +2,8 @@ language: node_js
 
 node_js:
   - "node"
+
+rvm:
+  - 2.2
+
```

Comment:
Since this is an Node base image, you'll need to probably add
```
after_install: bundle install
```
That way it will keep the default NPM install phase, then do the Ruby install after